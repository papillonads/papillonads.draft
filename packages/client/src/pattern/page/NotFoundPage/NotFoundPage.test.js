import React from 'react'
import notFoundPageDefault from './NotFoundPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<NotFoundPage />', () => {
  describe('Snaphot', () => {
    test('must match notFoundPageDefault', () => {
      expect(global.renderToJSON(notFoundPageDefault)).toMatchSnapshot()
    })
  })
})
