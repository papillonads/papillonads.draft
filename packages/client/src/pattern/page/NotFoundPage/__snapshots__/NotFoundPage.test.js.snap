// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> Snaphot must match notFoundPageDefault 1`] = `null`;
